sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -74,7 +80,10 @@ class DynamoDBOnlineStoreConfig(FeastConfigBaseModel):
     """Whether to read from Dynamodb by forcing consistent reads"""
 
     tags: Union[Dict[str, str], None] = None
-    """AWS resource tags added to each table"""
+    """Key-value pairs added to each feature-view"""
+
+    tag_aws_resources: StrictBool = False
+    """Add the feature-view tags to the underlying AWS dynamodb tables"""
```

Comment:
this feature isnt strictly necessary (though a good idea!) and results in a a few extra aws api calls per feature view, so make it opt in.